x-pack/metricbeat/module/aws/aws.go

```diff
@@ -58,6 +58,13 @@ func NewMetricSet(base mb.BaseMetricSet) (*MetricSet, error) {
 		return nil, err
 	}
 
+	var awsCred awscommon.ConfigAWS
+	err = base.Module().UnpackConfig(&awsCred)
```

Comment:
This should be done already, as `AWSConfig` is part of `Config`. Perhaps adding ``` `config:",inline"` ``` to that field fixes this?

Reply:
It works! Thanks!